sqlglot/dialects/bigquery.py

```diff
@@ -1242,15 +1242,15 @@ class Generator(generator.Generator):
             "end",
             "enum",
             "escape",
-            "except",
+            "except_",
```

Comment:
These shouldn't change, they are reserved keywords in the dialect and aren't used as Python identifiers. Same for other `RESERVED_KEYWORDS` modifications.

Reply:
they are all attributes now, they have to change